Userland/Games/2048/BoardView.cpp

```diff
@@ -24,6 +24,7 @@ void BoardView::set_board(Game::Board const* board)
     if (has_timer())
         stop_timer();
 
```

Comment:
(unrelated to code)
Please make the commit message an actual sentence, e.g. "2048: Add sliding animation"